docs/UserGuide.md

```diff
@@ -275,21 +275,21 @@ Example:<br>
 
 Filters tutor(s) from the list of matched tutors.
 
-Format: <br>`filter t [n/NAME] [g/GENDER] [q/QUALIFICATION]`
+Format: <br>`filter [n/NAME] [g/GENDER] [q/QUALIFICATION]`
```

Comment:
I think Xin Yan edited this part in her branch also? Can you confirm? @Lemonsr

Reply:
Yep I did!